packages/ui/src/molecules/OrderSummary/OrderSummary.test.tsx

```diff
@@ -0,0 +1,103 @@
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import OrderSummary from './OrderSummary'
+
+const ELEMENT_NOT_FOUND_MESSAGE = 'Unable to find an element by:'
+
+describe('OrderSummary', () => {
+  it('should have `data-store-order-summary` attribute', () => {
+    const { getByTestId } = render(<OrderSummary />)
+
+    expect(getByTestId('store-order-summary')).toHaveAttribute(
+      'data-fs-order-summary'
```

Comment:
Just noticed the test name is using the previous data attribute naming convention, e.g.:
```suggestion
  it('should have `data-fs-order-summary` attribute', () => {
    const { getByTestId } = render(<OrderSummary />)

    expect(getByTestId('store-order-summary')).toHaveAttribute(
      'data-fs-order-summary'
```